src/assets/curriculum/moving-straight-ahead/moving-straight-ahead.json

```diff
@@ -74,6 +74,7 @@
           "ordinal": 1,
           "title": "MSA 1.1 Walk Marathons",
           "subtitle": "Finding and Using Rates",
+          "disabled": ["GeometryMovableLineEquation"],
```

Comment:
Maybe a naive question, but what is the point of the line if the equation is not shown?  Perhaps a better question for the story author.

Reply:
In this particular problem, the student is asked to figure out the equation of the line that connects the points. If the line gives away the equation then there's nothing for the student to do.

Reply:
Makes sense!